apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -838,6 +886,99 @@ int _wt_parse_power(struct wt_options *opt, int argc, char *argv[])
 	return -1;
 }
 
+int _wt_parse_add_hostname(struct wt_options *opt, int argc, char *argv[])
+{
+ 	if (argc != 5) {
+		return -1;
+	}
+	opt->path = argv[3];
+	opt->hostname = argv[4];
+	return 0;		
+}
+
+int _wt_parse_stress5_client(struct wt_options *opt, int argc, char *argv[])
+{
+	if (argc < 4 || argc > 14) {
+		return -1;
+	}
+	opt->repeat_count = atoi(argv[3]);
+	opt->ssid = argv[4];
+	opt->auth_type = wt_get_auth_type(argv[5]);
+	if (opt->auth_type == WIFI_MANAGER_AUTH_OPEN || opt->auth_type == WIFI_MANAGER_AUTH_IBSS_OPEN) {
+		// case: open mode
+		opt->password = "";
+		opt->crypto_type = WIFI_MANAGER_CRYPTO_NONE;
+		return 0;
+	}
+
+	if (argc == 6) {
+		// case: unspecified security mode
+		opt->auth_type = WIFI_MANAGER_AUTH_UNKNOWN;
+		opt->crypto_type = WIFI_MANAGER_CRYPTO_UNKNOWN;
+		opt->password = argv[5];
+		return 0;
+	}
+
+	// case: security mode + password
+	if (opt->auth_type == WIFI_MANAGER_AUTH_UNKNOWN) {
+		return -1;
+	}
+
+	if (opt->auth_type == WIFI_MANAGER_AUTH_WEP_SHARED) {
```

Comment:
This code are used many places. So I made the parsing function for this. please refer _parse_security_str() in 
https://github.com/Samsung/TizenRT/pull/5311/files

Reply:
Changes Done as requested.